src/apply-names.cc

```diff
@@ -350,6 +352,18 @@ Result NameApplier::OnBrIfExpr(BrIfExpr* expr) {
   return Result::Ok;
 }
 
+Result NameApplier::OnBrOnNonNullExpr(BrOnNonNullExpr* expr) {
+  std::string_view label = FindLabelByVar(&expr->var);
+  UseNameForVar(label, &expr->var);
+  return Result::Ok;
+}
+
+Result NameApplier::OnBrOnNullExpr(BrOnNullExpr* expr) {
+  std::string_view label = FindLabelByVar(&expr->var);
+  UseNameForVar(label, &expr->var);
+  return Result::Ok;
```

Comment:
Unrelated to this PR but this seems to be a repeated pattern here.. i wonder if we could refactor/tempetize/macro somehow?

Reply:
True. I will do it.